lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -240,6 +241,7 @@ namespace :foreman_tasks do
 
     format = ENV['TASK_FORMAT'] || 'html'
     export_filename = ENV['TASK_FILE'] || "/tmp/task-export-#{Time.now.to_i}.#{format == 'csv' ? 'csv' : 'tar.gz'}"
+    skip_errors = ENV['SKIP_FAILED']
```

Comment:
How about you compare it with `"true"` here so `skip_errors` is always a boolean and not `nil` or `String`?

Reply:
Thank you so much for the "nitpicks" @adamruzicka !   I appreciate them.  I think they're all resolved i.e. applied now.